### PR TITLE
Fix path to confluence.git.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
   "dependencies": {
     "@google-cloud/storage": "^1.7.0",
     "foam2": "git://github.com/foam-framework/foam2.git#mdn-confluence",
-    "web-api-confluence-dashboard": "git://github.com/mdittmer/confluence.git#mdn",
+    "mdn-browser-compat-data": "git://github.com/mdn/browser-compat-data.git",
```
Comment (mdittmer) [inline package.json]: We need to keep this as a peer dependency to avoid a dependency cycle when BCD embeds us.

File: package.json [hunk 2]
```diff
@@ -15,11 +15,11 @@
   "dependencies": {
     "@google-cloud/storage": "^1.7.0",
     "foam2": "git://github.com/foam-framework/foam2.git#mdn-confluence",
-    "mdn-browser-compat-data": "git://github.com/mdn/browser-compat-data.git",
-    "web-api-confluence-dashboard": "git://github.com:mdittmer/confluence.git#mdn",
+    "mdn-browser-compat-data": "0.0.38",
+    "web-api-confluence-dashboard": "git://github.com/mdittmer/confluence.git#mdn",
```
Comment (mdittmer) [inline package.json]: Why `.com/` here and `.com:`below (in the `peerDependencies`)?

Comment (jpmedley) [reply package.json]: Because I submitted the PR prematurely.

File: package.json [hunk 3]
```diff
@@ -15,10 +15,11 @@
   "dependencies": {
     "@google-cloud/storage": "^1.7.0",
     "foam2": "git://github.com/foam-framework/foam2.git#mdn-confluence",
+    "mdn-browser-compat-data": "0.0.38",
     "web-api-confluence-dashboard": "git://github.com/mdittmer/confluence.git#mdn",
```
Comment (mdittmer) [inline package.json]: Now I'm confused. I thought this change was supposed to touch this line to "fix" the path to confluence.git.

**Pull request closed without merging**